### ISSUE TITLE
Accessibility Implementation

### ISSUE THREAD

**Issue by Lane** [EvictionLab/eviction-maps#127]
A few things to review for accessibility.  Feel free to add if you see anything missing:

- [x] Review components and add [ARIA attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Web_applications_and_ARIA_FAQ) where they are needed
- [x] Ensure proper `tabindex`'s have been added to components (see functional spec)
- [x] Review page and identify where `alt` text is needed.  Add alt text, or if none is available in the content matrix, add it so it can be adjusted / translated.
- [x] Run through the [WCAG checklist](https://www.wuhcag.com/wcag-checklist/) (Level A and Level AA) to ensure compliance
- [x] Test to ensure the tool is operable using only the keyboard.
- [x] Test out a screen reader to make sure the tool is operable.

**Comment by Lane** [comment]
Accessibility implementation delayed to allow time for NY Week revisions.

**Comment by pjsier** [comment]
Closing based on comments in #471
